widget/button.go

```diff
@@ -171,7 +177,10 @@ type Button struct {
 	OnTapped   func() `json:"-"`
 	HideShadow bool
 
-	hovered, tapped bool
+	focused  bool
```

Comment:
why 4 lines when 1 would do?

Reply:
I dont see how one line would do. There are different colors for all the combinations. But this should be simplified somehow. I will leave this for later.